201-front-door-api-management/azuredeploy.json

```diff
@@ -0,0 +1,1048 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for the resources."
+      }
+    },
+    "virtualNetwork": {
+      "type": "object",
+      "defaultValue": {
+        "name": "FrontDoorApimSampleVnet",
+        "addressPrefixes": [
+          "10.0.0.0/16"
+        ],
+        "subnets": [
+          {
+            "name": "ApiManagementSubnet",
+            "addressPrefix": "10.0.0.0/24",
+            "nsg": "FrontDoorApimSampleNsg"
+          }
+        ]
+      },
+      "metadata": {
+        "description": "Virtual Network configuration."
+      }
+    },
+    "workspaceName": {
+      "type": "string",
+      "defaultValue": "FrontDoorApimSampleLogAnalytics",
```

Comment:
does this need to be globally unique?

Reply:
Removed default value for workspaceName parameter.